src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/PatchUserIntTest.java

```diff
@@ -87,7 +88,7 @@ void patchUserShouldSucceedWhenProvidedWithValidValueForSubsetOfAllowableFields(
             .andExpect(jsonPath("$.full_name").value("Jimmy Smith"))
```

Comment:
there's a bit of a mix up going on with user_name and user_full_name - but i shall be resolving it with my ticket dmp-2033